script/write_config.py

```diff
@@ -76,6 +77,21 @@ def escape(s: str) -> str:
             s[i] = "\\s"
     return "".join(s)
 
+# Some values need to be "transformed", such as being converted from hex to decimal
+def transformValues(dOrig):
+    d = dOrig
+
+    cPath = os.path.expanduser("~/.config/kcminputrc")
+    if cPath in d:
+        for itemStr in list(d[cPath]):
+            if itemStr.startswith("Libinput/"):
+                item = d[cPath].pop(itemStr)
+                itemStrList = itemStr.split("/")
+                itemStrList[1] = str(int(itemStrList[1],16))
+                itemStrList[2] = str(int(itemStrList[2],16))
+                itemStr = "/".join(itemStrList)
+                d[cPath][itemStr] = item
```

Comment:
In python it is standard practice to use snake_case, not camelCase (unlike nix, java, C# etc). I think this should be the case here as well.

Reply:
This is my first real python code and I also happen to come from a java/c# background so I didn't know this, will change now